package.json

```diff
@@ -46,8 +46,6 @@
     "sinon": "^17.0.1",
     "stylelint": "^16.0.2",
     "stylelint-config-standard-scss": "^12.0.0",
-    "svgo": "^3.1.0",
```

Comment:
**Suggestion:** Can we please keep `svgo`? I use it quite often when working with SVG files and seems like it's [still fairly well-maintained](https://github.com/svg/svgo)

Reply:
Oh I wasn’t aware anyone was still using this script today. It hasn’t worked for me for some time.

If you think it’s still useful just for svg on its own, I can see if I can pick this apart.

Reply:
@reemhamz or do you mean you just use the regular svgo command line interface?